### ISSUE TITLE
Bug - Table - hoverable example invalid markup and selection not announced on navigation

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#7745]
**Describe the problem**
Followup to #7732 

The [hoverable selectable Table example](https://www.patternfly.org/v4/components/table#composable-row-click-handler-hoverable--selected-rows) throws the following warning in the dev environment:

`Warning: validateDOMNesting(...): <output> cannot appear as a child of <tbody>`

Additionally, due to the way screen readers navigate tables (by cell rather than by row), the row that is selected only gets announced when a row is selected via a cell. Using Tab to navigate between rows (as the table caption instructs) does announce whether the row is selected, but it only announces the `aria-label` and not the content of the row.

Some possible alternatives:

- Add a hidden input to each row, similar to the implementation seen in [Data list selectable rows example](https://www.patternfly.org/v4/components/data-list#selectable-rows). To avoid invalid markup, I believe this might require having an empty column similar to the table examples that use visible radio inputs.
- Add a hidden input to the first cell of a row, and add styling so that when the radio input has focus it looks like the row has focus.
- Move the current `output` element to the first `<td>` element of a row

**How do you reproduce the problem?**
Navigate the [Hoverable selectable table example](https://www.patternfly.org/v4/components/table#composable-row-click-handler-hoverable--selected-rows) with a screen reader

**Expected behavior**
Markup is valid, and users can be notified whether 

**Is this issue blocking you?**
List the workaround if there is one.

**Screenshots**
If applicable, add screenshots to help explain the issue.

**What is your environment?**

**What is your product and what release date are you targeting?**

**Any other information?**
